packages/core/src/actions/spawnChild.ts

```diff
@@ -210,6 +212,7 @@ export function spawnChild<
   spawnChild.src = src;
   spawnChild.input = input;
   spawnChild.syncSnapshot = syncSnapshot;
+  spawnChild.logger = logger;
```

Comment:
have you considered propagating this through the system? it feels like it could be cleaner